package.json

```diff
@@ -73,6 +74,7 @@
     "clean:build": "rimraf build/*",
     "copy:story-assets": "cpx \"src/stories/assets/**/*\" \"dist/stories/assets\"",
     "copy:readmes": "cpx \"src/components/**/README.md\" \"dist/components\"",
+    "create:svg:components": "svgcomp --dir=assets/Icons/optimized --output=src/components/Icon/icons",
```

Comment:
can the command just be `svg`?

Reply:
Sure! I thought I was following a pattern that you made :)